src/main/resources/standardTestCases/string_interpolation.test

```diff
@@ -0,0 +1,13 @@
+name: string_interpolation
+testFormat: workflowsuccess
+
+files {
+  wdl: string_interpolation/string_interpolation.wdl
+  inputs: string_interpolation/string_interpolation.inputs
```

Comment:
Hrm, this one appears to be a case where I *did* monkey with the workflow, and it's completely unclear to me why I did that.